packages/node_modules/nav-frontend-etiketter/src/index.tsx

```diff
@@ -11,19 +11,11 @@ const cls = (type, className) => classNames('etikett', className, {
     'etikett--info': type === 'info'
 });
 
-export interface EtikettProps {
+export interface EtikettProps extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement> {
```

Comment:
Hm, hva er forskjellen på å skrive:

```typescript
React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>
```
vs
```typescript
React.HTMLAttributes<HTMLDivElement>
```
🤔

Reply:
`React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>` 
er litt mer presis for typene til div hvis man ikke bruker klasse komponerer. men virker ikke med klasse komponerer
`React.HTMLAttributes<HTMLDivElement>`
er tilpasset klase komponenter

endrer til `React.HTMLAttributes<HTMLDivElement>`